NFIQ2/NFIQ2Algorithm/src/nfiq2/nfiq2_qualityfeatures_impl.cpp

```diff
@@ -102,7 +106,7 @@ NFIQ::QualityFeatures::Impl::getActionableQualityFeedback(
 				    ActionableQualityFeedbackThreshold_UniformImage ?
 				      true :
 				      false);
-			actionableQuality.push_back(fbUniform);
+			actionableMap["UniformImage"] = fbUniform;
```

Comment:
Use the constants (`ActionableQualityFeedbackIdentifier_UniformImage`) here and elsewhere
```suggestion
			actionableMap[ActionableQualityFeedbackIdentifier_UniformImage] = fbUniform;
```